Credits.md

```diff
@@ -0,0 +1,9 @@
+# Credits
+
+This project is made possible by the people and organizations listed in this document.
+
+## Contributors
+
+<img src="docs/images/id_pass_logo.svg" width="150" height="50">
```

Comment:
Could we add the name and a link under the logo like:
```markdown
[ID PASS](https://www.idpass.org/)
```
It would be easier for people who look at the text version of Markdown.